toml.md

```diff
@@ -161,9 +161,24 @@ In JSON land, that would give you the following structure:
 }
 ```
 
-Whitespace around dot-separated parts is ignored, however, best practice is to
+Whitespace around dot-separated parts is ignored. However, best practice is to
 not use any extraneous whitespace.
 
+```toml
+fruit.name = "banana"     # this is best practice
+fruit. color = "yellow"    # same as fruit.color
+fruit . flavor = "banana"   # same as fruit.flavor
+```
+
+Indentation is treated as whitespace. Tabs and spaces before keys are ignored.
```

Comment:
I would change the first sentence to "Indentation is ignored" – indentation IS whitespace (even in Python), so otherwise the sentence is just repeating obvious.

Reply:
It may be repeating the obvious to us, but not necessarily to Pythonistas and YAML punks. As I mentioned to @marzer, the repetition is deliberate. It's whitespace, it's ignored. Those two words always come together.

Call it overkill, but I'd rather have that in there now, going into v1.0.0, than leave any glimmer of doubt about what indentation is in TOML.